esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -423,7 +425,7 @@ impl<'d> Rtc<'d> {
         let rtc_cntl = unsafe { lp_aon() };
         rtc_cntl
             .store4()
-            .modify(|r, w| unsafe { w.bits(r.bits() | 1) });
+            .modify(|r, w| unsafe { w.bits(r.bits() | Self::RTC_DISABLE_ROM_LOG) });
```

Comment:
We might be better off adding this bit to the PAC, and that would also allow us to remove the masking of this bit in L508.

Reply:
My understanding is that these registers don't have actual functionality associated with them. Instead they're a way to pass data between the bootloaders and the application. Not sure if that's a PAC thing

Reply:
I guess that's technically correct, but if we use this as something fixed-function, we might as well make our lives a bit easier :) Not sure what the team thinks, though, it's just my opinion.

Reply:
I'd say it's just a register to store arbitrary data so probably it's better to keep it as is. `esp-hal` might not be the only consumer of the SVDs